build_tools/benchmarks/shark/run_shark.sh

```diff
@@ -14,26 +14,34 @@
 #        --sha <SHA of https://github.com/nod-ai/SHARK.git to pin to> \
 #        --pytest-regex e.g. "cpu", "cuda", "cuda and torch". \
 #        --driver e.g. "cpu", "cuda", "vulkan" \
+#        --save-version-info <true|false> \
+#        --use-local-iree <true|false> \
+#        --iree-source-dir <source directory of local iree> \
 #        --output-dir <local output dir> \
-#        --save-version-info <true|false>
 
 set -xeuo pipefail
 
 export SAVE_VERSION_INFO=false
 export BENCHMARK_REGEX="cpu"
 export DRIVER="cpu"
+export USE_LOCAL_IREE=false
+export IREE_SOURCE_DIR="$(pwd)"
 
 while [[ "$#" -gt 0 ]]; do
   case $1 in
-    -s|--sha) SHARK_SHA="$2"
+    --sha) SHARK_SHA="$2"
     shift;;
-    -r|--pytest-regex) BENCHMARK_REGEX="$2"
+    --pytest-regex) BENCHMARK_REGEX="$2"
     shift;;
     -d|--driver) DRIVER="$2"
     shift;;
+    --save-version-info) SAVE_VERSION_INFO="$2"
+    shift;;
+    --use-local-iree) USE_LOCAL_IREE="$2"
+    shift;;
     -o|--output-dir) SHARK_OUTPUT_DIR="$(pwd)/$2"
     shift;;
-    -v|--save-version-info) SAVE_VERSION_INFO="$2"
+    --iree-source-dir) IREE_SOURCE_DIR="$(pwd)/$2"
```

Comment:
This seems wrong if someone passed an absolute directory. I recommend either leaving it unchanged or using realpath, if for some reason later things need it to be absolute.

```suggestion
    --iree-source-dir) IREE_SOURCE_DIR="$(realpath $2)"
```

Reply:
Updated